src/fgimic/jfield.f90

```diff
@@ -376,8 +376,8 @@ subroutine jvector_plots(this,tag)
         end if
 
         ! put jvec information on vti file
-        if (this%grid%gauss.or.settings%prop) then
```

Comment:
Why did you change this line?

Reply:
Why did you not report this when filing the pull request? The feature "properties" has very recently been added, there is no test available yet.

Reply:
It's just too much work to explain every line of code.

Reply:
It was one line of code.